NEXID-1270 - Updated README.md in all apps (#79)
NexeraProtocol/nexeraid-examples: kyc_example/README.md

# KYC Example App

> This is the most basic KYC example flow that uses the SDK of NexeraID

Reviewer: ```suggestion
This is a simple KYC example flow, using the NexeraID Identity SDK.
```
